test/t8_forest_incomplete/t8_test_elements_interpolation.cxx

```diff
@@ -0,0 +1,309 @@
+#include <iostream>
+#include <t8.h>
+#include <t8_eclass.h>
+#include <t8_cmesh.h>
+#include <t8_cmesh/t8_cmesh_examples.h>
+#include <t8_forest.h>
+#include <t8_vec.h>
+#include <t8_schemes/t8_default/t8_default_cxx.hxx>
+#include <t8_forest/t8_forest_private.h>
+#include <t8_forest/t8_forest_iterate.h>
+#include "t8_cmesh/t8_cmesh_testcases.h"
+
+#include <t8_forest/t8_forest_partition.h>
+
```

Comment:
This is more of a visual test of the interpolation function and therefore should be moved to the examples folder. Can we implement a test that checks the correctness of the interpolation function?

Furthermore, please do not add your paths to the repository!

Reply:
Done. See `t8_gtest_iterate_replace.cxx` instead.